geode-core/src/main/java/org/apache/geode/distributed/internal/membership/gms/membership/GMSJoinLeave.java

```diff
@@ -376,10 +380,11 @@ public boolean join() {
               logger.debug("sleeping for {} before making another attempt to find the coordinator",
                   retrySleep);
               Thread.sleep(retrySleep);
-            }
+            } /* else { */
```

Comment:
we don't need the `else` anymore?

Reply:
2nd commit fixed that problem.  I had disabled it for some testing & forgot to re-enable the "else"

Reply:
my mistake, I was in a single commit view...